### DISCUSSION TITLE
Biome specific variants for Bounty Board block and structure.

### DISCUSSION THREAD

**Discussion by RubyRezal** [ejektaflex/Bountiful#203] [Mod Ideas]
It would be nice to see Bounty Board blocks and structures that better fit the theme or build style of each village type.

Examples:
![2021-12-05_11 21 20](https://user-images.githubusercontent.com/1816386/144755018-da18241e-5266-46da-a5c7-3056067b2512.png)
![2021-12-05_11 22 23](https://user-images.githubusercontent.com/1816386/144755028-10c1ecf4-ec1a-4971-85ee-0fc4b790304b.png)

In images: from left to right are the current bounty board structure (Plains style) then examples for Desert, Snowy Tundra, Savanna, and Taiga

**Comment by RubyRezal** [comment]
Example world save created in 1.18 with only Bountiful and dependencies installed.  Includes structure block files.

[Bountiful Structures Example.zip](https://github.com/ejektaflex/Bountiful-Fabric/files/7656513/Bountiful.Structures.Example.zip)

**Comment by ejektaflex** [comment]
Great idea - this is a PR in the Forge version [here](https://github.com/ejektaflex/Bountiful/pull/124). I should finish that up and merge, then implement it here sometime. Thanks!

**Comment by ejektaflex** [comment]
This has been implemented in 6.0.3!
